doc/stdlib/test.rst

```diff
@@ -258,6 +258,8 @@ arguments.
 
    Record a result to a testset. This function is called by the ``@testset`` infrastructure each time a contained ``@test`` macro completes, and is given the test result (which could be an ``Error``\ ). This will also be called with an ``Error`` if an exception is thrown inside the test block but outside of a ``@test`` context.
 
+.. function:: should_run(ts::AbstractTestSet, nruns::Int)
```

Comment:
Not quite, see https://github.com/JuliaLang/julia/blob/master/CONTRIBUTING.md#improving-documentation